src/main/java/com/jcabi/github/RtPull.java

```diff
@@ -159,6 +161,42 @@ public void merge(
     }
 
     @Override
+    public MergeState merge(
+        @NotNull(message = "message can't be NULL") final String msg,
+        final String sha) throws IOException {
+        final JsonObjectBuilder builder = Json.createObjectBuilder()
+            .add("commit_message", msg);
+        if (sha != null) {
```

Comment:
@sonOfRa You point is clear. I propose to make sha NonNull in this method and create an issue to improve previous one to return meaningful errors. What do you think?